src/app/AppErrorMessage.vue

```diff
@@ -4,13 +4,7 @@
     cta-is-hidden
   >
     <template #title>
-      <KIcon
-        class="mb-3"
-        icon="warning"
-        color="var(--black-500)"
-        secondary-color="var(--yellow-300)"
-        size="64"
-      />
+      <WarningIcon class="mb-3" />
```

Comment:
question: are you all planning to remove these utility classes in a later PR (to ensure they don't depend on the ones exported from Kongponents) or are these the ones you all define and plan to keep them?